themes/bootstrap3/less/components/cookie-consent/abstracts/light-color-scheme.less

```diff
@@ -48,4 +49,55 @@
     --cc-footer-bg: @panel-footer-bg;
     --cc-footer-color: @text-color;
     --cc-footer-border-color: @panel-default-border;
+/* #LESS> */
```

Comment:
This marker seems to be pointing in the wrong direction. Also, is it expected that there seem to be many more lines of SCSS than of LESS?

Reply:
Comments fixed. Not sure I understand the number of lines. I'm seeing 47 lines in both blocks.

Reply:
Sorry, I was just misreading the GitHub summary; looks good now!